versions.props

```diff
@@ -1,11 +1,13 @@
 com.fasterxml.jackson.core:jackson-databind = 2.18.1
 org.apache.commons:commons-lang3 = 3.17.0
 com.google.guava:guava = 33.3.1-jre
+com.palantir.baseline:gradle-baseline-java = 6.4.0
 
 # test dependencies
 com.netflix.nebula:nebula-test = 10.6.1
 org.assertj:assertj-core = 3.26.3
 org.junit.jupiter:* = 5.11.3
+org.codehaus.groovy:* = 3.0.21
```

Comment:
this is something I have seen before in other repos - if there is a dependency on gradle-baseline-java then there must also be a dependency on groovy setting the proper version or else test tasks get an error like "Failed to instantiate spec Conflicting module versions. Module [groovy-xml is loaded in version 3.0.21 and you are trying to load version 3.0.9"

Would be good to figure out how to avoid that, but it's not a problem to be solved in this repo specifically.